examples/rust/components/http-client/wadm.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: rust-http-client
+  annotations:
+    version: v0.0.1
+    description: "HTTP hello world demo in Rust, showing use of the server and client providers"
```

Comment:
Maybe include some information about the dog pictures and/or websites it uses?

Reply:
I don't know if that info needs to be here. I think of it more as a happy surprise when they look at the code